pkg/controller/bundlec_test/zz_plumbing_for_test.go

```diff
@@ -217,16 +203,16 @@ func (tc *testCase) run(t *testing.T) {
 		for _, reactor := range tc.scReactors {
 			scClientFake.AddReactor(reactor.verb, reactor.resource, reactor.reactor(t))
 		}
-		serviceClassInf := sc_v1b1inf.NewClusterServiceClassInformer(scClient, 0, cache.Indexers{})
-		infs = append(infs, serviceClassInf)
-		servicePlanInf := sc_v1b1inf.NewClusterServicePlanInformer(scClient, 0, cache.Indexers{})
-		infs = append(infs, servicePlanInf)
-		catalog, err = store.NewCatalog(serviceClassInf, servicePlanInf)
-		require.NoError(t, err)
 	}
 
-	crdInf := apiext_v1b1inf.NewCustomResourceDefinitionInformer(apiExtClient, 0, cache.Indexers{})
-	bundleInf := client.BundleInformer(smithClient.SmithV1(), meta_v1.NamespaceAll, 0)
+	plugins := make([]plugin.NewFunc, 0, len(tc.plugins))
+	for _, factory := range tc.plugins {
+		pluginInstance := factory(t)
+		plugins = append(plugins, func() (plugin.Plugin, error) { return pluginInstance, nil })
```

Comment:
shouldn't the func implementation invoke `factory(t)` every time? otherwise it's not `NewFunc` (semantically) if it always returns the same instance.

Reply:
ah, it's a test, probably fine.